tsconfig.json

```
@@ -21,6 +21,9 @@
   },
   "include": [
     "src",
-    "test"
+    "test",
+    "docs/.vitepress/*.ts",
```

```suggestion
    "docs/.vitepress",
```

This single line should be enough, right?